Phantomjs example does not work as configured

I am running intern 2.0 and followed the instructions in the phantom example and get connection refused errors. First, I run phantom on 4444 as per instructions:

``` bash
phantomjs --webdriver=4444
PhantomJS is launching GhostDriver...
[INFO  - 2014-07-15T22:37:37.759Z] GhostDriver - Main - running on port 4444
```

Then run the tests and see the error below:

``` bash
$ npm test

> phantomjs-example@0.1.0 test /Users/tom/code/intern-examples/phantomjs-example
> intern-runner config=tests/intern

Listening on 0.0.0.0:9000
Starting tunnel...
Error: connect ECONNREFUSED
  at errnoException  <net.js:904:11>
  at Object.afterConnect [as oncomplete]  <net.js:895:19>

TOTAL: tested 0 platforms, 0/0 tests failed; fatal error occurred
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0
```

I noticed that `intern.js` has:

``` js
tunnelOptions: {
    port: 4445
  }
```

So I tried again running phantom on 4445, and now I get a new error:

``` bash
$ npm test

> phantomjs-example@0.1.0 test /Users/tom/code/intern-examples/phantomjs-example
> intern-runner config=tests/intern

Listening on 0.0.0.0:9000
Starting tunnel...
UnknownError: [POST http://localhost:4445/wd/hub/session / {"desiredCapabilities":{"browserName":"phantom","name":"tests/intern","idle-timeout":60}}] An unknown server-side error occurred while processing the command.
  at Server._post  <node_modules/intern/node_modules/leadfoot/Server.js:62:9>
  at Server.createSession  <node_modules/intern/node_modules/leadfoot/Server.js:276:15>
  at Suite.setup  <node_modules/intern/runner.js:208:22>
  at callOnSuite  <node_modules/intern/lib/Suite.js:131:42>
  at call  <node_modules/intern/lib/Suite.js:182:13>
  at Suite.run  <node_modules/intern/lib/Suite.js:269:4>
  at <node_modules/intern/main.js:48:18>
  at <node_modules/intern/lib/util.js:324:19>
  at Array.forEach  <native>
  at Object.run  <node_modules/intern/main.js:47:16>

TOTAL: tested 0 platforms, 0/0 tests failed; fatal error occurred
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0
```

I then noticed that `intern.js` in this example is configured differently than in [the wiki](https://github.com/theintern/intern/wiki/Using-Intern-with-PhantomJS) - which says to use `browserName: 'phantomjs'` instead of `browserName: 'phantom'` so I tried using `phantomjs` but got basically the same error as above.

It seems like there are a couple of issues w/ the configuration in this example.

--- a/phantomjs-example/readme.md
+++ b/phantomjs-example/readme.md
@@ -2,6 +2,7 @@ phantomjs-example
 =============
 
 * Install PhantomJS
+* Ensure that the directory containing the phantomjs executable is in your PATH
 * Run `npm install`
-* Make sure you have PhantomJS WebDriver running with `phantomjs --webdriver=4444`
+* Start your local Selenium server
 * Run tests: `npm test`

--- a/phantomjs-example/tests/intern.js
+++ b/phantomjs-example/tests/intern.js
@@ -1,12 +1,10 @@
 define({
  environments: [
-    { browserName: 'phantom' }
+    { browserName: 'phantomjs' }
   ],
 
   suites: [ 'tests/test_example.js' ],
-  tunnelOptions: {
-    port: 4445
-  },
+  tunnel: 'NullTunnel',
 
   excludeInstrumentation: /./
 });

